### ISSUE TITLE
Height before creating the view

### ISSUE THREAD

**Issue by razvan-paul** [andreamazz/AMTagListView#59]
#3 This issue's solution tells that you can get the content size after you've created the tag list view, but I have a situation where I need it before. Can you please write a method which calculates that? I tried to look into your files, but there are a couple of interesting lines over there.

**Comment by andreamazz** [comment]
Hi @razvan-paul 
You need the content size before what? Before adding the tags?

**Comment by andreamazz** [comment]
I provide a delegate method for a single tag:
```objc
- (BOOL)tagList:(AMTagListView *)tagListView shouldAddTagWithText:(NSString *)text resultingContentSize:(CGSize)size;
```
I guess this could be expanded to multiple tags. That will eventually return the content size before adding the tags. Would that do?

**Comment by andreamazz** [comment]
Added in 996f0f5 a new delegate:
```objc
- (BOOL)tagList:(AMTagListView *)tagListView shouldAddTagsWithText:(NSArray *)tags resultingContentSize:(CGSize)size;
```

Let me know if that works for you.

**Comment by razvan-paul** [comment]
Perfect, just what I wanted. Thank you!

**Comment by andreamazz** [comment]
👍 
Released `1.0.2`
